### ISSUE TITLE
RadioButton - aria-checked attribute is missing

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#581]
### Describe the bug

When selecting a radio button, the “aria-checked” tag is not set. It is expected that the “aira-checked” is set when selecting:

`<input type="radio" class="p-radiobutton-input" value="Cheese" id="ingredient1" name="pizza" aria-checked="true" pc52="">`

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://v17.primeng.org/radiobutton

### Environment

PrimeNG v17  (17.18.15) is being used with Angular v17. The issue also replicable on https://v17.primeng.org/radiobutton

### Angular version

18.2.8

### PrimeNG version

v17 (LTS Only)

### Node version

22.5.1

### Steps to reproduce the behavior

1. Select a radio button element by clicking on label
2. Check the HTML-Code of the selected element "aria-checked = true" is excpeted

### Expected behavior

On selecting the radio button by clicking on label, the aria-checked property of the input[type="radio"] should set to "aria-checked = true"

> Migrated from `primefaces/primeng#17693` — originally by `@clude86` on 2025-02-17

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-10-15T00:20:11.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
